From 5a91c229fdd8f64ca2128b69512607fceff56142 Mon Sep 17 00:00:00 2001
From: Yong Yao <yong.yao@intel.com>
Date: Wed, 19 Feb 2014 11:01:28 +0800
Subject: Do the header copy in any case

No matter frameworks/base/core/jni/android_hardware_Camera.h valid or not do the header copy

Change-Id: I2be3792981a374964171462bb7556842a5ab096f
Signed-off-by: Yong Yao <yong.yao@intel.com>
---
 Android.mk |   18 +++++++++---------
 1 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/Android.mk b/Android.mk
index 81a46e0..e1c02df 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,5 +1,13 @@
-ifneq (,$(wildcard frameworks/base/core/jni/android_hardware_Camera.h))
 LOCAL_PATH:= $(call my-dir)
+################# COPY_HEADERS #######################
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := cameralibs
+LOCAL_COPY_HEADERS := \
+	include/intel_camera_extensions.h \
+	libacc/Ilibacc.h
+include $(BUILD_COPY_HEADERS)
+
+ifneq (,$(wildcard frameworks/base/core/jni/android_hardware_Camera.h))
 
 ################# MAKE_LIB ############################
 include $(CLEAR_VARS)
@@ -40,14 +48,6 @@ LOCAL_SRC_FILES := \
 	com/intel/camera/extensions/IntelCamera.java
 include $(BUILD_JAVA_LIBRARY)
 
-################# COPY_HEADERS #######################
-include $(CLEAR_VARS)
-LOCAL_COPY_HEADERS_TO := cameralibs
-LOCAL_COPY_HEADERS := \
-	include/intel_camera_extensions.h \
-	libacc/Ilibacc.h
-include $(BUILD_COPY_HEADERS)
-
 ################# burst capture sound ################
 include $(CLEAR_VARS)
 LOCAL_MODULE := fast_click.pcm
-- 
1.7.4.1

